subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractAndroidSantaTrackerSmokeTest.groovy

```diff
@@ -76,6 +103,17 @@ class AbstractAndroidSantaTrackerSmokeTest extends AbstractSmokeTest {
             }.build()
     }
 
+    protected BuildResult buildLocationMaybeExpectingWorkerExecutorAndConfigUtilDeprecation(File location, String agpVersion) {
+        return runnerForLocation(location, agpVersion,"assembleDebug")
+            .deprecations(SantaTrackerDeprecations) {
+                expectAndroidWorkerExecutionSubmitDeprecationWarning(agpVersion)
+                // TODO - this is here because AGP >=7.4 reads build/generated/source/kapt/debug at configuration time
```

Comment:
I read that comment as meaning something else than the code. I think the following suggestion clarifies it:
```suggestion
                // TODO - this is here because AGP 7.4.x reads build/generated/source/kapt/debug at configuration time
```